modules/ROOT/pages/sync-update-api-spec.adoc

```diff
@@ -0,0 +1,22 @@
+= Update the API Specification in Your Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+When necessary you can update the API specification in your Studio project.
```

Comment:
I would change this line as something like:

> When you import an API specification as a dependency, you can update the API specification version in your Studio project.

Only APIs imported as dependencies from a Maven repository, or Exhcange can be updated like this. 
API specs that are manually imported from the file system cannot be updated.